offix/src/main/java/org/aerogear/offix/Offix.kt

```diff
@@ -76,6 +110,24 @@ fun ApolloClient.enqueue(
     this.mutate(mutation).enqueue(apolloCallback)
 }
 
+/*
+
+ */
+fun getServerClientData(serverError: String): ServerClientData {
+    val json = JSONObject(serverError)
+    Log.d("Offix-Responsecallback1", " $detectedConflict")
+
+    val errorArray = json.optJSONArray("errors")
+    val errorObj1 = errorArray.optJSONObject(0)
```

Comment:
We might have to look into this later as to when we get more than 1 elements in the array of erros.

Reply:
Add a TODO here, which we can get back to later.

Reply:
Yes for sure.